app/controllers/api/works_controller.rb

```diff
@@ -21,9 +21,22 @@ def create
   end
 
   def update
-    work = Work.find(params[:id])
```

Comment:
I think you overrode some changes calvin made in his PR, can we add back those changes? they involve a schema restructuring